packages/dds/sequence/api-extractor.json

```diff
@@ -7,13 +7,5 @@
 		// This package has a root module named "sequence" which conflicts with the default naming here.
 		// TODO: remove this override once the base config has been updated to append `-untrimmed` globally.
 		"untrimmedFilePath": "<projectFolder>/dist/<unscopedPackageName>-untrimmed.d.ts"
-	},
```

Comment:
I just merged in some changes to this file you'll have to resolve conflicts with. My changes just removed the `dtsRollup` override block above, so should be straightforward to resolve.